packages/vapor/scss/components/badge.scss

```diff
@@ -72,6 +71,11 @@
         --background-color: var(--sienna-orange-20);
     }
 
+    &.mod-tag {
+        background: var(--white);
```

Comment:
```suggestion
        --background-color: var(--white);
```

Reply:
Does this work?

Reply:
It seemed to work, but I changed it to your (better) suggestion :)